packages/admin/src/pages/scheme/components/SchemeApplications.tsx

```diff
@@ -186,6 +188,7 @@ type SchemeApplicationsProps = {
   applicationForm: ApplicationFormSummary;
   applicationFormStats: FindApplicationFormStatsResponse;
   schemeVersion?: string;
+  editorOrPublisherEmail?: string;
```

Comment:
can this ever be false?

```suggestion
  editorOrPublisherEmail: string;
```